terraso_backend/locale/en/LC_MESSAGES/django.po

```diff
@@ -32,43 +32,59 @@ msgstr ""
 msgid "%(value)s is not allowed as a name"
 msgstr "%(value)s is not allowed as a name"
 
-#: terraso_backend/apps/core/models/groups.py:46
+#: terraso_backend/apps/core/models/groups.py:48
 msgid "Open"
 msgstr ""
 
-#: terraso_backend/apps/core/models/groups.py:47
+#: terraso_backend/apps/core/models/groups.py:49
 msgid "Closed"
 msgstr ""
 
-#: terraso_backend/apps/core/models/groups.py:167
+#: terraso_backend/apps/core/models/groups.py:50
+msgid "Restricted"
+msgstr ""
+
+#: terraso_backend/apps/core/models/groups.py:59
+msgid "Join"
+msgstr ""
+
+#: terraso_backend/apps/core/models/groups.py:60
+msgid "Invite"
+msgstr ""
+
+#: terraso_backend/apps/core/models/groups.py:61
+msgid "Both"
+msgstr ""
+
+#: terraso_backend/apps/core/models/groups.py:220
 msgid "Manager"
 msgstr ""
 
-#: terraso_backend/apps/core/models/groups.py:168
+#: terraso_backend/apps/core/models/groups.py:221
 msgid "Member"
 msgstr ""
 
-#: terraso_backend/apps/core/models/groups.py:175
+#: terraso_backend/apps/core/models/groups.py:228
 msgid "Approved"
 msgstr ""
 
-#: terraso_backend/apps/core/models/groups.py:176
+#: terraso_backend/apps/core/models/groups.py:229
 msgid "Pending"
 msgstr ""
 
-#: terraso_backend/apps/core/models/landscapes.py:74
+#: terraso_backend/apps/core/models/landscapes.py:77
 msgid "partnership_status.none"
 msgstr ""
 
-#: terraso_backend/apps/core/models/landscapes.py:75
+#: terraso_backend/apps/core/models/landscapes.py:78
 msgid "partnership_status.no"
 msgstr "No"
 
-#: terraso_backend/apps/core/models/landscapes.py:76
+#: terraso_backend/apps/core/models/landscapes.py:79
 msgid "partnership_status.in_progress"
 msgstr "In Progress"
 
-#: terraso_backend/apps/core/models/landscapes.py:77
+#: terraso_backend/apps/core/models/landscapes.py:80
 msgid "partnership_status.yes"
```

Comment:
Not sure why landscape translations are being updated, I didn't update them. If it is important to keep these files up to date we should add a check like a schema check to the CI.